src/js/me-i18n.js

```diff
@@ -40,6 +37,7 @@
     "use strict";
 
     var i18n = {
+        "default_language": 'en',
```

Comment:
I'll use only `default` since it's understood that is part of the `i18n` object for more simplicity